### PR TITLE
refresh configuration docs

### PR DESCRIPTION
**Pull request by dtrawins**
### 🧪 Checklist

- [ ] Unit tests added.
- [ ] The documentation updated.
- [ ] Change follows security best practices.
``

### CODE REVIEW

File: docs/models_repository_classic.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ ovms_docs_cloud_storage
 
 ```
 
-Traditional AI models perform data analysis in a single inference operation. They can be used over KServe API or TensorFlow API. 
+Classic models perform data analysis in a single inference operation. They can be served using the KServe API or TensorFlow API.
```
Comment (mzegla) [inline docs/models_repository_classic.md]:
```suggestion
Classic models perform data analysis in a single inference operation. They can be served using the KServe API. 
```

File: docs/pull_hf_models.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
 # OVMS Pull mode {#ovms_docs_pull}
 
-This document describes how to leverage OpenVINO Model Server (OVMS) pull feature to automate deployment configuration with Generative AI models. When pulling from [Hugging Face Hub](https://huggingface.co/) or when pulling GGUF model, no additional steps are required. However, when pulling models in Pytorch format, you have to install additional python dependencies when using baremetal execution so that optimum-cli is available for ovms executable or rely on the docker image `openvino/model_server:latest-py`. In summary you have 3 options:
+This document describes how to use the OpenVINO Model Server (OVMS) pull feature to automate deployment configuration for Generative AI models. When pulling models from [Hugging Face Hub](https://huggingface.co/) in IR or GGUF format, no additional setup is required. However, when pulling models in PyTorch format, you need additional Python dependencies on a bare-metal host so that `optimum-cli` is available to the OVMS executable, or you can use the `openvino/model_server:latest-py` Docker image. In summary, you have three options:
```
Comment (mzegla) [inline docs/pull_hf_models.md]: Are we ensuring `latest-py` tag being up to date? In the last release we did not push `latest-py`. 
Last `-py` image release was for 2026.1.

File: docs/pull_hf_models.md [hunk 3]
```diff
@@ -78,21 +78,21 @@ ovms --pull --source_model "unsloth/Llama-3.2-1B-Instruct-GGUF" --model_reposito
...
 
 Example for pulling `Echo9Zulu/phi-4-int4_asym-awq-ov`:
 
 ```text
-ovms --pull --source_model Echo9Zulu/phi-4-int4_asym-awq-ov --model_repository_path /models --model_name phi-4-int4_asym-awq-ov --target_device CPU --task text_generation 
+ovms --pull --source_model Echo9Zulu/phi-4-int4_asym-awq-ov --model_repository_path /models --model_name phi-4-int4_asym-awq-ov --target_device CPU --task text_generation
 ```
 
-> **Note:** These models aren't tested properly and their accuracy or performance may be low.
+> **Note:** These models are tested by OpenVINO team, and their accuracy or performance is not guaranteed.
```
Comment (mzegla) [inline docs/pull_hf_models.md]: Is that intentional? That would mean that models provided and tested by OV do not come with guaranteed quality. Shouldn't it be `are not tested`?

File: docs/starting_server.md [hunk 4]
```diff
@@ -314,55 +312,62 @@ docker run -d --rm -v <model_repository_path>:/models openvino/model_server:late
...
+
+> **Note:** If you want to add a model with a specific path, you can use the `--model_path` parameter:
 
 ```text
 docker run -d --rm -v <model_repository_path>:/models openvino/model_server:latest \
 --add_to_config --config_path <config_file_path> --model_name <name> --model_path <model_path>
 ```
 
-*Note:* Use relative or absolute paths. `config_path` is relative to the current folder. `model_path` is relative to the config file. With `model_repository_path`, `model_name` represents a folder relative to model repository path. 
+> **Note:** Use relative or absolute paths. `config_path` is relative to the current folder. `model_path` is relative to the config file. With `model_repository_path`, `model_name` represents a folder relative to the model repository path.
```
Comment (mzegla) [inline docs/starting_server.md:336]: For `model_path` I guess it depends if we talk about model path in config.json, or model_path in a single model mode provided from the CLI right? But I guess it's fine if in that section we are talking about `add_to_config` case.

File: docs/pull_hf_models.md (hunk 3 above)
Comment (mzegla) [reply docs/pull_hf_models.md]: @dtrawins 

```suggestion
> **Note:** These models are NOT tested by OpenVINO team, and their accuracy or performance is not guaranteed.
```

**Review by mzegla — APPROVED** [review]

File: docs/models_repository_classic.md [hunk 5]
```diff
@@ -80,7 +78,25 @@ It can map the input and output keys to the appropriate tensors. This extra mapp
...
+ovms --add_to_config --config_path config.json --model_name model1 --target_device GPU --batch_size 1 --model_path models/model1
+```
+
+Models can also be removed from the configuration file:
+```text
+ovms --remove_from_config --config_path config.json --model_name model1
+```
+
+All models can be started with the following command:
+```
```
Comment (przepeck) [inline docs/models_repository_classic.md]: missing `text`

File: docs/pull_hf_models.md [hunk 6]
```diff
@@ -1,27 +1,27 @@
...
+There is a special OVMS mode that pulls a model from Hugging Face without starting the service. It is triggered by the `--pull` parameter. The application exits after the model is downloaded. Without `--pull`, the model is deployed and the server starts.
 
 ::::{tab-set}
 :::{tab-item} With Docker
 :sync: docker
 **Required:** Docker Engine installed
 ```text
-docker run $(id -u):$(id -g) --rm -v <model_repository_path>:/models:rw openvino/model_server:weekly --pull --source_model <model_name_in_HF> --model_repository_path /models --model_name <external_model_name> --target_device <DEVICE> [--gguf_filename SPECIFIC_QUANTIZATION_FILENAME.gguf] --task <task> [TASK_SPECIFIC_PARAMETERS]
+docker run --user $(id -u):$(id -g) --rm -v <model_repository_path>:/models:rw openvino/model_server:latest-gpu --pull --source_model <model_name_in_HF> --model_repository_path /models --model_name <external_model_name> --target_device <DEVICE> [--gguf_filename SPECIFIC_QUANTIZATION_FILENAME.gguf] --task <task> [TASK_SPECIFIC_PARAMETERS]
```
Comment (przepeck) [inline docs/pull_hf_models.md]: don't we want to use docker image `openvino/model_server:2026.3-gpu`?

File: docs/pull_hf_models.md [hunk 7]
```diff
@@ -41,11 +41,11 @@ Example for pulling `OpenVINO/Phi-3-mini-FastDraft-50M-int8-ov`:
 **Required:** Docker Engine installed
 
 ```text
-docker run $(id -u):$(id -g) --rm -v <model_repository_path>:/models:rw openvino/model_server:latest --pull --source_model "OpenVINO/Phi-3-mini-FastDraft-50M-int8-ov" --model_repository_path /models --model_name Phi-3-mini-FastDraft-50M-int8-ov --task text_generation
+docker run --user $(id -u):$(id -g) --rm -v <model_repository_path>:/models:rw openvino/model_server:latest-gpu --pull --source_model "OpenVINO/Phi-3-mini-FastDraft-50M-int8-ov" --model_repository_path /models --model_name Phi-3-mini-FastDraft-50M-int8-ov --task text_generation
 ```
 :::
 
-:::{tab-item} On Baremetal Host
+:::{tab-item} On Bare-metal Host
```
Comment (przepeck) [inline docs/pull_hf_models.md:48]: we always use "Baremetal" in our docs

File: docs/models_repository_classic.md (hunk 5 above)
Comment (dtrawins) [inline docs/models_repository_classic.md]:
```suggestion
```text
```

File: docs/pull_hf_models.md (hunk 6 above)
Comment (dtrawins) [inline docs/pull_hf_models.md]:
```suggestion
docker run --user $(id -u):$(id -g) --rm -v <model_repository_path>:/models:rw openvino/model_server:2026.3-gpu --pull --source_model <model_name_in_HF> --model_repository_path /models --model_name <external_model_name> --target_device <DEVICE> [--gguf_filename SPECIFIC_QUANTIZATION_FILENAME.gguf] --task <task> [TASK_SPECIFIC_PARAMETERS]
```

File: docs/pull_hf_models.md (hunk 7 above)
Comment (dtrawins) [reply docs/pull_hf_models.md:48]: the version with dash is more correct

**Review by przepeck — APPROVED** [review]

File: docs/models_repository_classic.md [hunk 8]
```diff
@@ -80,7 +78,24 @@ It can map the input and output keys to the appropriate tensors. This extra mapp
...
+ovms --add_to_config --config_path config.json --model_name model1 --target_device GPU --batch_size 1 --model_path models/model1
+```
+
+Models can also be removed from the configuration file:
+```text
+ovms --remove_from_config --config_path config.json --model_name model1
+```
+
+All models can be started with the following command:
+ovms --config_path config.json --rest_port 8000 --port 9000
```
Comment (dtrawins) [inline docs/models_repository_classic.md]:
```suggestion
```text
ovms --config_path config.json --rest_port 8000 --port 9000
```
